cdap-ui/app/cdap/components/AbstractWidget/MemorySelectWidget/index.js

```diff
@@ -38,3 +38,8 @@ export default function MemorySelectWidget({ ...props }) {
 
 MemorySelectWidget.propTypes = WIDGET_PROPTYPES;
 MemorySelectWidget.defaultProps = DEFAULT_WIDGET_PROPS;
+MemorySelectWidget.getWidgetAttributes = () => {
+  return {
+    size: { type: 'Size', required: false },
```

Comment:
Where is this type defined?

Reply:
I have defined it here! https://github.com/cdapio/cdap/blob/c77629cc5271a0b2e42c01c5a1059979d406a9bf/cdap-ui/app/cdap/components/ConfigurationGroup/WidgetWrapper/index.tsx#L88